docs/tables/crowdstrike_user.md

```diff
@@ -0,0 +1,29 @@
+# Table: crowdstrike_user
+
+Users in the Falcon system.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  first_name,
+  last_name,
+  uid
+from
+  crowdstrike_user
+```
+
+### List users with specific roles
```

Comment:
What roles would users typically want to check for most? Are there any high privilege roles?